docs/explanation/charm/charm-compatibility.md

```diff
@@ -0,0 +1,41 @@
+(explanation-charm-compatibility)=
+
+# Landscape Server charm integration compatibility
+
+The [Landscape Server charm](https://charmhub.io/landscape-server) can be deployed with Juju when integrated with at least the following charms:
+
+- `haproxy`
+- `postgresql`
+- `rabbitmq-server`
+
+For a recommended charm bundle configuration and more information about deploying Landscape Server with Juju, see {ref}`How to install Landscape Server with Juju <how-to-juju-installation>`.
+
+Outside of charm base (e.g., `ubuntu@24.04`) or architecture (e.g., `amd64`) incompatibility, there can be breaking changes to the interfaces of the required charms that make them incompatible for integration with specific or all revisions and channels of the Landscape Server charm.
+
+```{tip}
+Learn more about [Juju integrations](https://canonical.com/juju/integrations).
+```
+
+## K8s Operators
+
+Landscape Server is currently only distributed as a machine (VM) charm and cannot be directly integrated with any version of K8S charm operators like the HAProxy K8s operator or the Charmed PostgreSQL K8s operator.
```

Comment:
```suggestion
Landscape Server is currently only distributed as a machine (VM) charm and cannot be directly integrated with any version of K8s Charmed Operators, such as the HAProxy K8s operator or the Charmed PostgreSQL K8s operator.
```
I looked at [their docs](https://ubuntu.com/kubernetes/charmed-k8s/docs/operator-charms) for the "K8s Charmed Operators" formatting. I guess it's capitalized, didn't know that :)